### DISCUSSION TITLE
Flatpak & flatseal

### DISCUSSION THREAD

**Discussion by Derisis13** [realmazharhussain/gdm-settings#149] [Q&A]
Following #83 lead me to suspect that there's a misconfigured/unset permission in the flatpak version found on flathub. The appimage provided there can set my login background, the flatpak cannot.
What permission do I have to give the program with flatseal to be able to change backgrounds? Or is it not a matter of permission, but because the flathub version is outdated/the provided appimage is a pre-release?

**Comment by realmazharhussain** [comment]
As far as I know, the Flathub version has access to everything it needs to access.

You can try out Flatpak and AppImage of the latest alpha (v3.alpha.0) from its [release page](https://github.com/realmazharhussain/gdm-settings/releases/tag/v3.alpha.0) and see if they work.

Also, looking at the terminal output helps a lot when looking for the root of an issue.

**Reply by dallasrevolt** [reply]
I'm also having issues with Flatpak version on Ubuntu 22.04. Some settings do apply (hide the Accesibility menu and show the weekday), but the theme and the background do not. Here's the output:
```
dallas@Ubuntu:~$ flatpak run io.github.realmazharhussain.GdmSettings -v

(gdm-settings:2): Gtk-WARNING **: 10:19:19.728: Theme parser error: gtk.css:5688:3-9: No property named "height"

(gdm-settings:2): Gtk-WARNING **: 10:19:19.954: Theme parser error: gtk.css:5688:3-9: No property named "height"

(gdm-settings:2): Gtk-WARNING **: 10:19:19.973: Theme parser error: gtk.css:5688:3-9: No property named "height"
INFO: Application Version    = 3.3
INFO: Operating System       = Ubuntu 22.04.3 LTS
INFO: PackageType            = Flatpak
INFO: TEMP_DIR               = /run/user/1000/app/io.github.realmazharhussain.GdmSettings
INFO: HOST_DATA_DIRS         = /usr/local/share:/usr/share
INFO: ShellGresourceFile     = /usr/share/gnome-shell/gnome-shell-theme.gresource
INFO: UbuntuGdmGresourceFile = /usr/share/gnome-shell/gdm3-theme.gresource
INFO: pkexec version 0.105
INFO: Backing up default shell theme …
INFO: Applying GResource settings for Ubuntu …

(gdm-settings:2): Gtk-CRITICAL **: 10:20:02.633: _gtk_css_corner_value_get_x: assertion 'corner->class == &GTK_CSS_VALUE_CORNER' failed

(gdm-settings:2): Gtk-CRITICAL **: 10:20:02.633: _gtk_css_corner_value_get_y: assertion 'corner->class == &GTK_CSS_VALUE_CORNER' failed
```
